.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
## Your File Names

![image](https://user-images.githubusercontent.com/93039359/228144370-80c33293-ad81-4f8d-8411-6b87190546d1.png)

## Requirements
![image](https://user-images.githubusercontent.com/93039359/228144882-29d8873d-3e38-496e-a253-d098ec5c85aa.png)


- [ ] It would be very great if you rename your files style.css to index.css and script.js to index.js to follow the requirements of the project 😊 Once you start working on javascript project then you will always see your entry point file starting with index whether its a javascript file or html or css 🤔